manim/mobject/graph.py

```diff
@@ -769,7 +769,7 @@ def add_edges(
         ----------
 
         edges
-            The edge (as a tuple of vertex identifiers) to be added. If a non-existing
+            A comma separated list of edges (as a tuple of vertex identifiers) to be added. If a non-existing
```

Comment:
Same as above, I think it best rewording to not include "A comma separated list" to avoid confusion